application/common/application_resource.cc

```diff
@@ -14,7 +14,7 @@ namespace xwalk {
 namespace application {
 namespace {
 const base::FilePath::StringType WGT_LOCALE_DIRECTORY =
-    FILE_PATH_LITERAL("locales");
+    FILE_PATH_LITERAL("locales/xwalk");
```

Comment:
You're supposed to check the code you are changing instead of blindingly replacing everything. This looks like the implementation of the wgt spec, so it has nothing to do with this pull request.